src/components/05_pages/Content/ConfirmationDialog.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from 'emotion';
+
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Slide from '@material-ui/core/Slide';
+
+const styles = {
+  dialogActionName: css`
+    text-transform: lowercase;
+  `,
+};
+
+const ConfirmationDialog = ({
+  action,
+  actions,
+  checked,
+  contentList,
+  dialogVisibility,
+  handleClose,
+  executeAction,
+}) => {
+  const actionLabel = actions
+    .filter(({ attributes: { id } }) => id === action)
+    .map(({ attributes: { label } }) => label)
+    .shift();
+  return (
+    <Dialog
+      open={dialogVisibility}
+      TransitionComponent={props => <Slide direction="up" {...props} />}
+      onClose={handleClose}
+    >
+      <DialogTitle>
+        Are you sure you want to{' '}
```

Comment:
The text is a little bit confusing right now: ```Are you sure you want to make content sticky these content items?```. I'm wondering whether we could use something like "Are you sure you want to apply the \"make content sticky\" action on these content items?"```